### ISSUE TITLE
TOC generator: broken bullet points (`input*` instead of `  *`)

### ISSUE THREAD

**Issue by christian-korneck** [RickStrahl/MarkdownMonster#389]
In Markdown Monster `1.12` (on Win7 SP1 x64, .NET 4.7.1 32bit, OS language: English, OS locale: German), when I try to create/embedd a TOC (table of content) from the Outline view, all bullet points below nesting level 1 are created incorrectly. Instead of two spaces (`  `) or a tab for the indentation, the string `input` is being used.

##### Source Markdown Document:

```
# Captains: At a glance
# Captains
## Kirk
 ### Details
## Archer
## Picard
```

##### Expected Result from TOC generator:
```
<!-- Start Document Outline -->
* [Captains: At a glance](#captains-at-a-glance)
* [Captains](#captains)
  * [Kirk](#kirk)
    * [Details](#details)
  * [Archer](#archer)
  * [Picard](#picard)
<!-- End Document Outline -->
```

##### Actual Result from TOC generator:
```
<!-- Start Document Outline -->
* [Captains: At a glance](#captains-at-a-glance)
* [Captains](#captains)
input* [Kirk](#kirk)
inputinput* [Details](#details)
input* [Archer](#archer)
input* [Picard](#picard)
<!-- End Document Outline -->
```

As you can see, instead of two spaces per nesting level in front of a bullet point, the string `input` is being inserted.

**Comment by RickStrahl** [comment]
Oh man - looks like that one is a failure in one of the support libraries (mine). 

Fixed: 167e3d7522c85803ff059724717f8a061b25f386

both for MM as well as fixing the underlying bug in [Westwind.Utilities](https://github.com/RickStrahl/Westwind.Utilities)

**Comment by christian-korneck** [comment]
thanks
